tests/Support/IntegrationTestHelpers.php

```
@@ -354,6 +362,76 @@ public static function connectToDatabase(string $type, DatabaseServer $server, s
+    }
+
+    private static function firebirdIsqlBinary(): string
+    {
+        exec('command -v isql-fb 2>/dev/null', $output, $exitCode);
+
+        return $exitCode === 0 && isset($output[0]) ? $output[0] : 'isql';
+    }
+
+    public static function canRunFirebirdIntegration(): bool
+    {
+        exec('command -v isql-fb 2>/dev/null', $output, $exitCode);
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$output'. (undefined)

(UnusedLocalVariable)